prac_08/convert_miles_km.py

```diff
@@ -0,0 +1,41 @@
+"""
+Rhys Bush - Miles to Kilometres Converter
+Expected time: 20 minutes.
+Actual time: 40 minutes.
+"""
+
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.app import StringProperty
+
+MILES_TO_KM_FACTOR = 1.609
+
+
+class MilesToKilometresConverter(App):
+    output_km = StringProperty()
```

Comment:
This class is missing class docstring and also method docstrings

Reply:
That is true - I tend to forget commenting.